apis/openstack-nova/src/main/java/org/jclouds/openstack/nova/v2_0/NovaApi.java

```diff
@@ -457,4 +457,12 @@ ImageApi getImageApiForZone(
    @Delegate
    Optional<? extends ConsolesApi> getConsolesExtensionForZone(
          @EndpointParam(parser = RegionToEndpoint.class) String zone);
+
+    /**
+     * Provides access to interface Type features.
+     */
+    @Delegate
+    Optional<? extends InterfaceApi> getInterfaceApiForZone(
+            @EndpointParam(parser = RegionToEndpoint.class) String zone);
```

Comment:
Should be `Optional<InterfaceApi> getInterfaceApi(@EndpointParam(parser = RegionToEndpoint.class) String region);`